Framework/Algorithms/src/WienerSmooth.cpp

```diff
@@ -385,8 +389,9 @@ WienerSmooth::smoothSingleSpectrum(API::MatrixWorkspace_sptr inputWS,
  *   centres are used.
  * @return :: A pair of start x and end x.
  */
-std::pair<double, double> WienerSmooth::getStartEnd(const MantidVec &X,
-                                                    bool isHistogram) const {
+std::pair<double, double>
+WienerSmooth::getStartEnd(const Mantid::HistogramData::HistogramX &X,
+                          bool isHistogram) const {
```

Comment:
This could have been made nicer by passing `Points`. The `isHistogram` flag would then not be necessary. However, that would add significant overhead since only two of the points are needed, so I agree that this should be kept as it is 👍 .